app/models/state_file_w2.rb

```diff
@@ -96,9 +96,8 @@ def state_tax_group_xml_node
     xml_template.at(:LocalWagesAndTipsAmt).content = local_wages_and_tips_amt
     xml_template.at(:LocalIncomeTaxAmt).content = local_income_tax_amt
     xml_template.at(:LocalityNm).content = locality_nm
-
+    delete_blank_nodes(xml_template)
     result = xml_template.at(:W2StateTaxGrp)
-    delete_blank_nodes(result)
-    result.to_xml
+    result.nil? ? "" : result.to_xml
   end
```

Comment:
We now remove nodes from the overall document object rather than just the root node. This means that we can remove the root node